Orange/tests/test_table.py

```diff
@@ -2019,7 +1772,7 @@ def test_delete_rows(self):
     def test_clear(self):
         self.table.clear()
         self.assertEqual(len(self.table), 0)
-        for i in self.table:
+        for _ in self.table:
```

Comment:
It is not the same. **For** will call `__iter__` (or `__getitem__` if `__iter__` is not defined). **If** will call `__bool__`.

But we could (should?) test both.